hotsos/defs/scenarios/openstack/nova/bugs/lp2012284.yaml

```diff
@@ -0,0 +1,18 @@
+checks:
+  has_2012284:
+    input:
+      path: 'var/log/nova/nova-compute.log'
+    expr: '([\d-]+) ([\d:]+).\d{3} .+Failure prepping block device: gi.repository.GLib.GError: g-io-error-quark: Error opening directory ./usr/share/osinfo.: Permission denied'
```

Comment:
So now I see I commented the opposite in your other patch - let's wait for Ed to confirm which is correct.